src/app/store/machine/slice.ts

```diff
@@ -977,44 +993,111 @@ const machineSlice = createSlice({
     exitRescueModeStart: statusHandlers.exitRescueMode.start,
     exitRescueModeSuccess: statusHandlers.exitRescueMode.success,
     fetch: {
-      prepare: (requestId: string) => ({
+      prepare: (requestId: string, params?: FetchParams) => ({
         meta: {
           model: MachineMeta.MODEL,
           method: "list",
+          nocache: true,
```

Comment:
We should probably find a way to keep the cache, atm the machine list is always requested at least twice (I have a ticket that's related to this https://github.com/canonical-web-and-design/maas-ui/issues/4274)

Reply:
I suspect we'll end up with just one request as the other requests will change to using the count endpoint.